benchexec/tools/FuSeBMC.py

```diff
@@ -0,0 +1,79 @@
+# This file is part of BenchExec, a framework for reliable benchmarking:
+# https://github.com/sosy-lab/benchexec
+#
+# SPDX-FileCopyrightText: 2007-2020 Dirk Beyer <https://www.sosy-lab.org>
+#
+# SPDX-License-Identifier: Apache-2.0
+
+import os
+import benchexec.util as util
+import benchexec.tools.template
+import benchexec.result as result
+
+
+class Tool(benchexec.tools.template.BaseTool):
+    """
+    This class serves as tool adaptor for ESBMC (http://www.esbmc.org/)
+    """
+
+    REQUIRED_PATHS = ["esbmc", "esbmc-wrapper.py", "my_instrument"]
```

Comment:
Having the executable that is returned from `executable()` in this is redundant.